### PR TITLE
Crdcdh 160 updates

### PR DESCRIPTION
**Pull request by mtangmt**
This implements the listDataSubmissions page.
This does not fulfill all the requirements.
Pending: 
- Get full list of organizations to filter by.

The create a submission button is functional enough for us, but pending some backend changes to fulfill the entire requirements.

### CODE REVIEW

**Review by amattu2 — CHANGES_REQUESTED** [review]
I just did a cursory review of the Type definitions, we have some duplicates in the codebase because multiple features were developed simultaneously. Please review and adjust as necessary. There may be other duplicates so please double check for any I missed.

File: src/graphql/listSubmissions.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export type Response = {
+  listSubmissions: {
+    total: number;
+    submissions: Omit<Submission, "submitterID"
+    | "modelVersion" | "bucketName" | "rootPath" | "history">[];
+  };
+};
+
+// type Submission {
```
Comment (amattu2) [inline src/graphql/listSubmissions.ts:35]: Please confirm if we need these comments for something or if they are an accident.

**Comment by amattu2** [comment]
Adding hold-merge label until the organizations list is updated in the BE and implemented in the FE. Feel free to remove when ready.

File: src/types/Submissions.d.ts [hunk 2]
```diff
@@ -18,7 +18,27 @@ type DataSubmission = {
   updatedAt: string; // # ISO 8601 date time format with UTC or offset e.g., 2023-05-01T09:23:30Z
 };
 
-type DataSubmissionStatus = "New" | "In Progress" | "Submitted" | "Released" | "Withdrawn" | "Rejected" | "Completed" | "Archived" | "Canceled";
+type SubmissionHistoryEvent = {
```
Comment (Alejandro-Vega) [inline src/types/Submissions.d.ts:21]: This is the same thing as the line below: `type DataSubmissionHistoryEvent = HistoryBase<SubmissionStatus>;`
See HistoryBase type for more details.

File: src/types/Submissions.d.ts [hunk 3]
```diff
@@ -18,7 +18,27 @@ type DataSubmission = {
...
+  status: SubmissionStatus; // # [New, In Progress, Submitted, In Review, Approved, Rejected]
+  reviewComment: string; // # if applicable
+  dateTime: string; // # YYYY-MM-DDTHH:MM:SS format
+  userID: string;
+};
+
+type SubmissionStatus = "New" | "In Progress" | "Submitted" | "Released" | "Withdrawn" | "Rejected" | "Completed" | "Archived" | "Canceled";
+
+type HistoryEvent = {
```
Comment (Alejandro-Vega) [inline src/types/Submissions.d.ts:30]: This already exists in src/types/Application.d.ts as: `type HistoryEvent = HistoryBase<ApplicationStatus>;`

File: src/types/Submissions.d.ts [hunk 4]
```diff
@@ -18,7 +18,27 @@ type DataSubmission = {
...
+  status: ApplicationStatus;
+  reviewComment?: string;
+  dateTime: string; // YYYY-MM-DDTHH:MM:SSZ format
+  userID: number;
+};
+
+type Applicant = {
```
Comment (Alejandro-Vega) [inline src/types/Submissions.d.ts:37]: This already exists in src/types/Application.d.ts does this have a use for data submissions?

**Pull request merged**